test/core/bootstrapers.spec.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-env mocha */
+'use strict'
+
+const chai = require('chai')
+const dirtyChai = require('dirty-chai')
+const expect = chai.expect
+chai.use(dirtyChai)
+const IPFS = require('../..')
+
+/*
+ * These tests were graciously made for lgierth, so that he can test
+ * the WebSockets Bootstrappers easily :)
+ */
+console.log('=>', process.env.TEST_BOOTSTRAPERS)
+if (process.env.TEST_BOOTSTRAPERS) {
```

Comment:
@dignifiedquire how can I get env vars to be 'browserified' to the browser context?